### PR TITLE
Support MinimumTlsVersion property for Redis Cache cmdlets

### LINKED ISSUE #9601: Get-AzRedisCache does not return minimumTlsVersion where "az redis show" does
## Description

The Redis properties exposed with `Get-AzRedisCache` are not consistent with what `az redis show` returns. In particular, I'm interested in `minimumTlsVersion`, which I can't find a way to return with the PS module.

## Steps to reproduce

```powershell
# Observe properties returned with PS module
Get-AzRedisCache -Name $name -ResourceGroupName $rg

# Observe properties returned with az cli
az redis show --name $name --resource-group $rg
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      6.2.0
PSEdition                      Core
GitCommitId                    6.2.0
OS                             Microsoft Windows 10.0.18362 
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}        
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

## Module versions

```powershell
ModuleType Version    Name
---------- -------    ----
Script     1.1.0      Az.RedisCache
```

### PR DESCRIPTION
**Pull request by MECHANDR**
Added `MinimumTlsVersion` parameter in `New-AzRedisCache` and `Set-AzRedisCache` cmdlets. Also, added `MinimumTlsVersion` in the output of `Get-AzRedisCache` cmdlet.  Fixes #9601 #9923

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/RedisCache/RedisCache/Commands/SetAzureRedisCache.cs [hunk 1]
```diff
@@ -57,6 +58,10 @@ public class SetAzureRedisCache : RedisCacheCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "The number of shards to create on a Premium Cluster Cache.")]
         public int? ShardCount { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Specify the TLS version required by clients to connect to cache.")]
+        [ValidateSet(TlsStrings.OneFullStopZero, TlsStrings.OneFullStopOne, TlsStrings.OneFullStopTwo)]
```
Comment (msJinLei) [inline src/RedisCache/RedisCache/Commands/SetAzureRedisCache.cs]: Use argument completer here.

File: src/RedisCache/RedisCache/Commands/NewAzureRedisCache.cs [hunk 2]
```diff
@@ -63,6 +64,10 @@ public class NewAzureRedisCache : RedisCacheCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "The number of shards to create on a Premium Cluster Cache.")]
         public int? ShardCount { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Specify the TLS version required by clients to connect to cache.")]
+        [ValidateSet(TlsStrings.OneFullStopZero, TlsStrings.OneFullStopOne, TlsStrings.OneFullStopTwo)]
```
Comment (msJinLei) [inline src/RedisCache/RedisCache/Commands/NewAzureRedisCache.cs]: Use argument completer here.

**Comment by MECHANDR** [comment]
Can someone help with build issues of running checks. PR doesn't have any changes which would have caused these error.
```C:\Program Files\dotnet\sdk\3.0.100\NuGet.targets(123,5): error : Unable to load the service index for source https://www.myget.org/F/azure-powershell/api/v3/index.json. [D:\a\1\s\tools\BuildPackagesTask\Microsoft.Azure.Build.Tasks\Microsoft.Azure.Build.Tasks.csproj] [D:\a\1\s\build.proj]
C:\Program Files\dotnet\sdk\3.0.100\NuGet.targets(123,5): error : Response status code does not indicate success: 503 (Service Unavailable). [D:\a\1\s\tools\BuildPackagesTask\Microsoft.Azure.Build.Tasks\Microsoft.Azure.Build.Tasks.csproj] [D:\a\1\s\build.proj]
D:\a\1\s\build.proj(103,5): error MSB3073: The command "dotnet publish D:\a\1\s\tools/BuildPackagesTask/Microsoft.Azure.Build.Tasks/Microsoft.Azure.Build.Tasks.csproj -c Debug -f netstandard2.0" exited with code 1.
##[error]Error: The process 'C:\Program Files\dotnet\dotnet.exe' failed with exit code 1
##[error]Dotnet command failed with non-zero exit code on the following projects : ```

**Pull request merged**